grunt/config/less.js

```diff
@@ -4,12 +4,17 @@ module.exports = function (grunt, options) {
             options:{
                 baseUrl: '<%= sourcedir %>',
                 mandatory: [
-                    '<%= sourcedir %>core/less/**/*.less'
+                    '<%= sourcedir %>core/less/*.less',
```

Comment:
is there any need for these to be listed individually? they produce the same thing, only the original is shorter.

Reply:
Nope, updating.